react-native/debugging/debug-two-ios-apps-simultaneously.mo.md

```diff
@@ -0,0 +1,35 @@
+# [MO] Debug two iOS apps simultaneously
+
+## Owner: [Xavier Lefèvre](https://www.github.com/xavierlefevre)
+
+## Last update date: 29th of January 2018
```

Comment:
OK pour moi, mais comment s'assurer que ça soit le cas partout et que ça reste à jour.

On a fait ça sur les  std déprécié, c'est ptet plus maintenable que la date : https://bamtech.gitbook.io/dev-standards/react-native/setup/mo-setup-and-deploy-new-project-to-staging-with-hockeyapp-deprecated